### PR TITLE
fix(ui5-menu): make close event non-bubbling

### PR DESCRIPTION
**Pull request by ilhan007**
Fixes: https://github.com/SAP/ui5-webcomponents/issues/10041

### CODE REVIEW

File: packages/main/cypress/specs/LitKeyFunction.cy.ts [hunk 1]
```diff
@@ -37,14 +37,14 @@ describe("Lit HTML key function for #each", () => {
...
-			.eq(3)
-			.should("contain.text", "USA")
-			.should("have.attr", "selected");
+		// cy.get("@items")
+		// 	.eq(0)
+		// 	.should("contain.text", "<empty>")
+		// 	.should("not.have.attr", "selected");
+
+		// cy.get("@items")
+		// 	.eq(3)
```
Comment (nnaydenow) [inline packages/main/cypress/specs/LitKeyFunction.cy.ts:46]: do we need to comment this? it is passing to me.

File: packages/main/cypress/specs/base/Events.cy.ts [hunk 2]
```diff
@@ -156,12 +176,26 @@ describe("Event bubbling", () => {
 
 		cy.get("@dialog")
 			.then(dialog => {
+				dialog.get(0).addEventListener("close", cy.stub().as("dialogClosed"));
```
Comment (nnaydenow) [inline packages/main/cypress/specs/base/Events.cy.ts:179]: I'm not sure about naming the aliases and when they are linked to two different stubs. Can we rename one of them to dialogClosed2?

File: packages/main/cypress/specs/base/Events.cy.ts [hunk 3]
```diff
@@ -156,12 +176,26 @@ describe("Event bubbling", () => {
...
+				select.get(0).addEventListener("close", cy.stub().as("selClosed"));
+			});
+
+		cy.get("@menu")
+			.then(menu => {
+				menu.get(0).addEventListener("close", cy.stub().as("menuClosed"));
+			});
+
+		cy.get("@button")
+			.then(button => {
```
Comment (nnaydenow) [inline packages/main/cypress/specs/base/Events.cy.ts:194]: this is redundant since you call ui5Open later in the test.

**Review by nnaydenow — CHANGES_REQUESTED** [review]

File: packages/main/cypress/specs/LitKeyFunction.cy.ts (hunk 1 above)
Comment (ilhan007) [reply packages/main/cypress/specs/LitKeyFunction.cy.ts:46]: locally never fails, but fails very often on github

**Review by nnaydenow — APPROVED** [review]

**Pull request merged**